vpdq/python/tests/test_vpdq_hash.py

```diff
@@ -28,15 +33,24 @@
     "pattern-sd-with-small-logo-bar",
 ]
 
-project_dir = Path(__file__).parents[3]
-hash_folder = project_dir / SAMPLE_HASH_FOLDER
-video_folder = project_dir / SAMPLE_VIDEOS
+PROJECT_DIR = Path(__file__).parents[3]
+HASH_FOLDER = PROJECT_DIR / SAMPLE_HASH_FOLDER
+VIDEO_FOLDER = PROJECT_DIR / SAMPLE_VIDEOS
 test_hashes = {}
 sample_hashes = {}
 
+TEST_FILES = []
+for fileStr in glob.iglob(f"{VIDEO_FOLDER}/**/*.mp4", recursive=True):
+    file = Path(fileStr)
+    if not (VIDEO_FOLDER / f"{file.name}").is_file():
+        print(f"Video file {file.name} doesn't exist. Skipping.")
+        continue
+    TEST_FILES.append(file)
+assert len(TEST_FILES) > 0
```

Comment:
Sure, and then I will also remove the hardcoded test names to allow more flexibility for adding videos.

The regtest might also break if more videos are added to tmk, but I don't have the energy to fix that right now and that was the same as the old one.

I wish the sample videos were in the base directory instead of in tmk the repo but it is what it is.

Also, is there a reason why Git LFS wasn't used to store the videos?

Reply:
> but I don't have the energy to fix that right now and that was the same as the old one.

No worries, that sounds fine to me

> Also, is there a reason why Git LFS wasn't used to store the videos?

I would guess because we didn't know about Git LFS 😅. I've been soapboxing internally about not putting large files into git into the past.